transformer_engine/pytorch/csrc/userbuffers/CMakeLists.txt

```
@@ -26,6 +27,11 @@ set_source_files_properties(userbuffers.cu
                             COMPILE_OPTIONS "$<$<COMPILE_LANGUAGE:CUDA>:-maxrregcount=64>")
 set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} --expt-relaxed-constexpr")
 set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -O3")
+if(DEFINED ENV{NVTE_WITH_MNNVL})
```

It would be nice to disable with `NVTE_WITH_MNNVL=0`:
```suggestion
set(NVTE_WITH_MNNVL "$ENV{NVTE_WITH_MNNVL}")
if(NVTE_WITH_MNNVL)
```